src/core/channels/presence_channel.ts

```diff
@@ -84,9 +90,7 @@ export default class PresenceChannel extends PrivateChannel {
         if (removedMember) {
           this.emit('pusher:member_removed', removedMember);
         }
-        break
-      default:
```

Comment:
Shouldn't we have  a `default` case, at least to log that something unexpected was received?

Reply:
Elsewhere, we don't log/expose anything to do with unhandled `pusher_internal` events. I can see a case for calling `super` in case any `internal` events are handled at the `Channel` level, but in this case I think it's consistent.